polaris/ocean/model/mpaso_to_omega.yaml

```diff
@@ -1,4 +1,12 @@
-map:
+variables:
+  temperature: Temp
+  salinity: Salt
+  tracer1: Debug1
+  tracer2: Debug2
+  tracer3: Debug3
+  ssh: SshCell
```

Comment:
I'm not actually sure this is correct.  We will see once Omega is actually writing output.